backend/scraper/scrape_walmart.py

```diff
@@ -49,7 +62,7 @@ def parse_product_link(self, url, session):
 
         # Product Title
         try:
-            product_title = soup.find(class_='prod-ProductTitle font-normal').get_text()
+            product_title = soup.find(class_='prod-ProductTitle prod-productTitle-buyBox font-bold').get_text()

```

Comment:
Why was this change necessary? Has Walmart changed the layout, or was it wrong to begin with?

Reply:
It was working up until now, so I checked the web page and the class name was changed.